.github/actions/build-sign-publish-chainlink/action.yml

```diff
@@ -38,6 +38,15 @@ inputs:
     description: When set to the string boolean value of "true", the resulting build image will be signed
     default: "false"
     required: false
+  cosign-private-key:
+    description: The private key to be used with cosign to sign the image
+    required: false
+  cosign-password:
+    description: The password to decrypt the cosign private key needed to sign the image
+  sign-method:
```

Comment:
should we prefix all cosign related inputs with `cosign`?

Reply:
Good idea, let me draft up another PR, that uses a newer version of cosign and does this.